lectures/deep.js

```diff
@@ -274,16 +279,8 @@ add(slide('Prior knowledge',
   bulletedText(stmt('Intuition: try to match a pattern in image')),
 _).leftHeader('[figure from Andrej Karpathy]'));
 
-add(slide(null,
-  stmt('Fully-connected'),
-  parentCenter(xtable(
-    vectorBox(null, 10), thickRightArrow(50),
-    vectorBox(null, 5), thickRightArrow(50),
-    vectorBox(null, 7),
-  _).center().margin(10)),
-  pause(),
-  stmt('Convolutional: each depth column produced from localized region (in height/width)'),
-  parentCenter(image('images/cnn.jpeg').width(600)),
+add(slide('Convolutional layers',
```

Comment:
I think the contrast is helpful

Reply:
What do you mean contrast?

Reply:
Contrast fully-connected with convolutions